### DISCUSSION TITLE
APT repository meta-data dropped the "grafana" identifier in `Origin` and `Label`

### DISCUSSION THREAD

**Discussion by suominen** [grafana/grafana#59870] [General]
Hi,

Recently `apt` came up with the following:

```
E: Repository 'https://packages.grafana.com/oss/deb stable InRelease' changed its 'Origin' value from 'grafana stable' to '. stable'
E: Repository 'https://packages.grafana.com/oss/deb stable InRelease' changed its 'Label' value from 'grafana stable' to '. stable'
```

This does not seem like an intentional change. Could someone have a look, please?

Kind regards,
\+ Kimmo

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
